code/espurna/config/hardware.h

```diff
@@ -4211,6 +4211,54 @@
     #define SENSOR_ENERGY_UNITS         ENERGY_KWH
     #define SENSOR_POWER_UNITS          POWER_WATTS
 
+// -----------------------------------------------------------------------------
+// Kogan Smarter Home Plug with Energy Meter (Australia)
+// Product code: KASPEMHA
+// https://www.kogan.com/au/buy/kogan-smarterhome-smart-plug-energy-meter/
+// Reflashing from original Tuya firmware
+// to thirdparty firmware like espurna by:
+// https://github.com/ct-Open-Source/tuya-convert
+// -----------------------------------------------------------------------------
+
+#elif defined(KOGAN_SMARTER_HOME_PLUG_W_POW)
+
+    // Info
+    #define MANUFACTURER                "KOGAN"
+    #define DEVICE                      "SMARTER_HOME_PLUG_POW"
```

Comment:
`defined(...string....)` should match `MANUFACTURER` + `_` + `DEVICE`
We need to keep one style, either `PLUG_W_POW` or `PLUG_POW` at the end. Whichever you prefer.
ofc pio ini and arduino header should match